src/mrinufft/operators/base.py

```diff
@@ -695,6 +695,7 @@ def _adj_op(self, coeffs, image):
             coeffs2 = coeffs
         self.raw_op.adj_op(coeffs2, image)
 
+    @with_numpy_cupy
     def data_consistency(self, image_data, obs_data):
```

Comment:
should be `@with_numpy`

Reply:
This will be handled in #140 as we cannot test this right now.